src/Layouts/NO_QWERTY.ts

```diff
@@ -0,0 +1,43 @@
+import {LayoutMap} from './Layout';
+
+export const id = 'NO QWERTY';
```

Comment:
Weeelll.. NO is the ISO3166 two-letter country code for Norway, so it is actually a sensible choice. The problem is that it's also the word 'NO'. Say you get an Azerbaijanian layout, Would you call it 'AZ' or Azerbaijanian? Maybe we should make one more example, just to clarify that this is actually the country code, and not "Not a QWERTY keyboard mapping"?